### ISSUE TITLE
Debug flags

### ISSUE THREAD

**Issue by fortunac** [draperlaboratory/cbat_tools#136]
Look into the Z3 flags to see if we can print out some debug info such (e.g. are we bit blasting when solving a constraint?)

**Comment by philzook58** [comment]
Also one can print statistics from the solver after the solve
```
Printf.printf " statistics : %s \n %!" (Z3.Statistics.to_string (Z3.Solver.get_statistics solver) ); 
```

**Comment by gltrost** [comment]
Incorporating Landmarks has been reiterated in issue #169.
No obvious solution for showing bit blasting was found. 
The other main issues brought up have been resolved by #183.
